zarr/zarr-in-practice.ipynb

```diff
@@ -8,11 +8,38 @@
     "\n",
```

Comment:
<p>I would remove the pangeo forge catalog as I'm not sure if those datasets are actively maintained.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/172/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>